notices.json

```diff
@@ -15,5 +15,23 @@
         "actionParam": "https://mattermost.com/download?inapp-notice=true#mattermostApps"
       }
     }
+  },
+  {
+    "id": "server_upgrade_v5.29",
```

Comment:
@reflog can you confirm this usage of symbols `_` and `.` is acceptable for a notice ID?

Reply:
@esethna - anything can be used for IDs. The only limitation is length - 26 characters (as all our IDs)